classes/03class/exercises/c03-docker01/starkmatt/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:latest
```

Comment:
It's not a best practice to use the latest tag, since it can change and you won;t be able to be sure what version of the alpine image you're using to build your image, which can bring potential compatibility issues or even security reasons.

Instead of using the latest tag, just pin the version that currently corresponds to latest at this moment.

Great job on getting the proper version and a very small image!

Reply:
Thanks @kikobr82 !
I've made the change to the Dockerfile